satpy/readers/fci_l1c_nc.py

```diff
@@ -204,21 +204,31 @@ def end_time(self):
         """Get end time."""
         return self.filename_info['end_time']
 
+    def get_channel_measured_group_path(self, channel):
+        """Get the channel's measured group path."""
+        if self.filetype_info['file_type'] == 'fci_l1c_hrfi':
+            channel += '_hr'
+        measured_group_path = 'data/{}/measured'.format(channel)
+
+        return measured_group_path
+
     def get_segment_position_info(self):
         """Get the vertical position and size information of the chunk (aka segment) for both 1km and 2km grids.
```

Comment:
Should this be extended for 500m in addition to 1km and 2km grids?

Also: I know this was not added by this PR, but what is the "vertical position"?  That's something sounders would have.  I suppose here we mean "latitude"?

Reply:
Good point yes, thanks. 
By vertical position I mean the position of the chunk (segment) inside the full-disc in the vertical image direction, i.e. the number of the start and end row. I can clarify this in the docstring.

Reply:
I know that's what you mean, and I know that in 2-dimensional Cartesian coordinates the x-coordinates is sometimes called horizontal and the y-coordinate is sometimes called vertical, but in an Earth observation context I find it not the best choice of words, because of the competing definition of vertical to mean "along the local gravity direction".  The satellite height is a vertical coordinate.

Reply:
Maybe `y_position` then?

Reply:
Updated now, I tried to clarify the concept in the docstring a bit more, see https://github.com/pytroll/satpy/pull/2287/files#diff-5e6ace3bb73933d0b7fb85a692590ac424b905677f665f84a6e73f2dc3cf7097R224-R229